src/vorta/application.py

```diff
@@ -171,3 +173,38 @@ def _alert_missing_borg(self):
         msg.setInformativeText(self.tr("Vorta was unable to locate a usable Borg Backup binary."))
         msg.setStandardButtons(QMessageBox.Ok)
         msg.exec()
+
+    def react_to_log(self, mgs, context):
+        """
+        Trigger Vorta actions based on Borg logs. E.g. repo lock.
+        """
+        msgid = context.get('msgid')
+        if msgid == 'LockTimeout':
+            profile = BackupProfileModel.get(name=context['profile_name'])
+            repo_url = context.get('repo_url')
+            msg = QMessageBox()
+            msg.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+            msg.setText(
+                self.tr(
+                    f"The repository at {repo_url} might be in use by another computer. Override it and continue?"))
+            msg.accepted.connect(lambda: self.break_lock(profile))
```

Comment:
uuh, this almost sounds like recommending to break it.

Reply:
I thought the same actually. Here another take inspired by https://borgbackup.readthedocs.io/en/stable/usage/lock.html

> The repository at {repo_url} might be in use elsewhere. Only proceed if you are certain no other Borg process on any machine is accessing the repository. Continue to break lock?

Reply:
Yeah, that is much better.

Could be also:

The repository at {repo_url} might be in use elsewhere. Only break the lock if you are certain no other Borg process on any machine is accessing the repository. Abort or break the lock?

Reply:
That will work too. I played a bit with the buttons, icon and text to make it more scary.

![Screen Shot 2021-02-17 at 22 33 13](https://user-images.githubusercontent.com/3916435/108218948-2e2e6380-7170-11eb-883c-332530c1ef9c.jpg)

Reply:
#867